src/Explore/ScoreIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+
+import { Box } from '../Primitives'
+
+function ScoreIndicator({ score }) {
+  const [text, setText] = useState(score.toFixed(3))
+
+  const handleHoverStart = () => {
+    setText('Relevancy')
+  }
+
+  const handleHoverEnd = () => {
+    setText(score.toFixed(3))
+  }
+
+  return (
+    <Box
+      sx={{
+        position: 'relative',
+        width: 96,
+        background: 'linear-gradient(to right, #bb4677, #82b8e4)',
+        borderRadius: 8,
+        marginLeft: 1,
+        overflow: 'hidden',
```

Comment:
If you put `z-index: 0` here, you shouldn't need to increase the `z-index` on the navigation bar.